### PR TITLE
Add a new ShiftStack team member

### PR DESCRIPTION
**Pull request by pierreprinetti**

### CODE REVIEW

**Review by mandre — APPROVED** [review]
/lgtm

**Review by Fedosin — APPROVED** [review]
/lgtm

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 aliases:
   openstack-approvers:
+    - EmilienM
     - Fedosin
     - adduarte
     - iamemilio
     - mandre
     - pierreprinetti
   openstack-reviewers:
+    - EmilienM
```
Comment (EmilienM) [inline OWNERS_ALIASES:12]: thanks!

**Review by kikisdeliveryservice — APPROVED** [review]

**Pull request merged**